core/src/test/java/org/apache/iceberg/avro/TestReadProjection.java

```diff
@@ -37,10 +37,11 @@
 import org.junit.rules.TemporaryFolder;
 
 public abstract class TestReadProjection {
+
   protected abstract Record writeAndRead(String desc,
-                                         Schema writeSchema,
-                                         Schema readSchema,
-                                         Record record) throws IOException;
+      Schema writeSchema,
+      Schema readSchema,
+      Record record) throws IOException;
```

Comment:
Can you revert the non-functional changes here? This one and the whitespace one above.